node/src/Pos/Block/Network/Retrieval.hs

```diff
@@ -89,75 +89,105 @@ retrievalWorkerImpl SendActions {..} =
         queue        <- view (lensOf @BlockRetrievalQueueTag)
         recHeaderVar <- view (lensOf @RecoveryHeaderTag)
         logDebug "Waiting on the block queue or recovery header var"
+        -- Reading the queue is a priority, because it sets the recovery
+        -- variable in case the header is classified as alternative. So if the
+        -- queue contains lots of headers after a long delay, we'll first
+        -- iterate over them and set recovery variable to the latest one, and
+        -- only then we'll do recovery.
```

Comment:
I think GH is getting confused here, as this is coming from [this PR](https://github.com/input-output-hk/cardano-sl/commit/b46bae4a209ef19ce9c5c3f6b6c5d69ad594bd23#diff-1b9bae52a122b1082d4116552252e6c7) that we backported into `master` [here](https://github.com/input-output-hk/cardano-sl/pull/2225/files).